models/Hike.js

```diff
@@ -0,0 +1,42 @@
+const { Model, DataTypes } = require('sequelize');
+const sequelize = require('../config/connection');
+
+class Hike extends Model {}
+
+Hike.init(
+  {
+    //id, time, description, user_id, trail_id
+    id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        primaryKey: true,
+        autoIncrement: true,
+    },
+    time: {
+        type: DataTypes.INTEGER,
+        allowNull: false,        
```

Comment:
I think we can allow the time and description to be null. That way a user can log a completed hike even if they didn't record their time or don't want to write a description.